lib/pseudable.rb

```diff
@@ -2,14 +2,27 @@
 module Pseudable
 
   def pseuds_to_add=(pseud_names)
-    pseud_names.split(',').reject {|name| name.blank?}.map {|name| name.strip}.each do |name|
+    names = pseud_names.split(',').
+                        reject { |name| name.blank? }.
+                        map { |name| name.strip }
+    names.each do |name|
       possible_pseuds = Pseud.parse_byline(name)
       if possible_pseuds.size > 1
-        possible_pseuds = Pseud.parse_byline(name, :assume_matching_login => true)
+        possible_pseuds = Pseud.parse_byline(name, assume_matching_login: true)
+      end
+      pseud = possible_pseuds.first
+      user = pseud.user
+      if pseud.nil?
+        errors.add(:base, 
+                   ts("We couldn't find the pseud {{name}}.", name: name))
+      elsif user.banned?
```

Comment:
I don't know what official style says however:
 
   if pseud.nil?
       errors.add(:base, ts("We couldn't find the pseud {{name}}.", name: name))
       return
 end
if user.banned?
       errors.add(:base, s("{{name}} has been banned and cannot be listed as a co-creator", name: name)
       return
end

Is easier for simple brains like me to understand.